### ISSUE TITLE
Image is not show

### ISSUE THREAD

**Issue by officialrajdeepsingh** [avitorio/outstatic#155]
### Provide environment information

**Version of available packages:**

1. next: `13.5.6`
2. outstatic `0.0.41`
3. react `18.2.0`
4. react-dom 18.2.0`
5. @tailwindcss/typography `0.5.10`
6. @types/react-dom 18.2.14`       
7. eslint `8.52.0`                   
8. remark `15.0.1`                   
9. typescript `5.2.2`                
10. @types/node `20.8.8`              
11. autoprefixer `10.4.16`            
12. eslint-config-next `13.5.6`       
13. remark-html `16.0.1`              
14. @types/react `18.2.31`            
15. dayjs `1.11.10`                   
16. postcss `8.4.31`                  
17. tailwindcss `3.3.4`

### Describe the Bug

When outstatic CMS convert markdown file in `/outstatic/content/posts` folder. It adds `>` in end of image URL.

```markdown
![Follower](https://cdn.pixabay.com/photo/2023/10/27/17/04/dahlia-8345799_1280.jpg> "an-image-title")
```

[https://github.com/officialrajdeepsingh/outstatic-demo](https://github.com/officialrajdeepsingh/outstatic-demo)

### Expected Behavior

Remove the `>` in markdown image.

```markdown
![Follower](https://cdn.pixabay.com/photo/2023/10/27/17/04/dahlia-8345799_1280.jpg "an-image-title")
```

### Link to reproduction - Issues with a link to complete (but minimal) reproduction code will be addressed faster

https://github.com/officialrajdeepsingh/outstatic-demo

### To Reproduce

[https://github.com/officialrajdeepsingh/outstatic-demo/blob/main/outstatic/content/posts/15-facts-you-never-knew-about-photography.md?plain=1](https://github.com/officialrajdeepsingh/outstatic-demo/blob/main/outstatic/content/posts/15-facts-you-never-knew-about-photography.md)

![Image URL](https://github.com/avitorio/outstatic/assets/32416397/b918fb10-f6da-4489-b43c-8c0e4e93636e)

**Comment by avitorio** [comment]
@officialrajdeepsingh Thanks for the issue! I'll look it up.

**Comment by avitorio** [comment]
Hi @officialrajdeepsingh, one question. Have you added cdn.pixabay.com to your allowed domains on the next.config.js?

https://nextjs.org/docs/app/api-reference/components/image#domains

**Comment by officialrajdeepsingh** [comment]
Now, the image URL issue is fixed in outstatic 1.0.3.
